scripts/generate/templates/SUBSCRIPTION_MANAGEMENT/vanilla.js.template

```diff
@@ -76,33 +78,38 @@ function Add(root, api) {
 }
 
 // 'Create' mode should create a new selling plan, and add the current product to it
+// [Shopify admin renders this mode inside an app chrome container]
 function Create(root, api) {
   const data = api.data;
   const sessionToken = api.sessionToken;
-  const {close, done, setPrimaryAction, setSecondaryAction} = api.container;
+  const {close, done} = api.container;
 
-  setPrimaryAction({
-    content: 'Create plan',
-    onAction: async () => {
-      // Get a fresh session token before every call to your app server.
-      const token = await sessionToken.getSessionToken();
+  const pageProps = {
+    title: `Create subscription plan`,
```

Comment:
Should all the templates use this title instead of `Create subscription`?

Reply:
Oops, yep! I wasn't sure which is better, I think it's more descriptive with the word **plan**, updating unless there are objections!